### PR TITLE
test

### PR DESCRIPTION
**Pull request by robbanthony**

### CODE REVIEW

**Comment by webhat** [comment]
I think you accidentally pushed this changes upstream. :)

File: app/views/preorder/_copyright.html.erb [hunk 1]
```diff
@@ -1 +1 @@
-<p class="pull-right">© You. Made by the guys at <a href="https://lockitron.com">Lockitron</a>. MIT License.</p>
+<p class="pull-right">© Robb Anthony, 2013. MIT License.</p>
```
Comment (webhat) [inline app/views/preorder/_copyright.html.erb:1]: You know you should leave credits in for the original creator, otherwise it violates the MIT license.

**Comment by robbanthony** [comment]
Whoops! I did. My bad.

On Oct 29, 2013, at 12:57 PM, "Daniel W. Crompton" notifications@github.com wrote:

I think you accidentally pushed this changes upstream. :)

File: app/views/preorder/_copyright.html.erb (hunk 1 above)
Comment (robbanthony) [inline app/views/preorder/_copyright.html.erb:1]: Right. Thank you!

**Comment by Jarred-Sumner** [comment]
Haha it's okay.

File: app/views/preorder/_copyright.html.erb (hunk 1 above)
Comment (webhat) [inline app/views/preorder/_copyright.html.erb:1]: Glad to help.

Comment (davidmabelle) [inline app/views/preorder/_copyright.html.erb:1]: Please unsubscribe.

From:  "Daniel W. Crompton" notifications@github.com
Reply-To:  lockitron/selfstarter
<reply+p-7298798-a9177ec4e773d842d37f801da0aead12cee5312c-997504@reply.githu
b.com>
Date:  Tuesday, October 29, 2013 1:34 PM
To:  lockitron/selfstarter selfstarter@noreply.github.com
Subject:  Re: [selfstarter] test (#62)

In app/views/preorder/_copyright.html.erb:

> @@ -1 +1 @@
> -<p class="pull-right">© You. Made by the guys at <a
> href="https://lockitron.com">Lockitron</a>. MIT License.</p>
> +<p class="pull-right">© Robb Anthony, 2013. MIT License.</p>
> Glad to help.

‹
Reply to this email directly or view it on GitHub
https://github.com/lockitron/selfstarter/pull/62/files#r7298798 .

**Pull request closed without merging**